Probesys/bileto
# new: Allow a user to change its locale

Changes proposed in this pull request:

- Add a `locale` column to users
- Load the locale from the session `_locale` parameter
- Set the session `_locale` parameter during login to the value of `user.locale`
- Add a locale select box on the preferences page

How to test the feature manually:

1. Login and go to the preferences page
2. Change the locale and check the strings are translated correctly

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
